paper/paper.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Augur: a bioinformatics toolkit for phylogenetic analyses of human pathogens"
+papersize: a4
+tags:
+  - Python
+  - bioinformatics
+  - molecular biology
+  - human pathogens
+  - evolutionary biology
+date: 5 November 2020
+bibliography: paper.bib
+---
+
+# Summary and statement of need
+
+The analysis of human pathogens requires a diverse collection of bioinformatics tools.
+These tools include standard genomic and phylogenetic software and custom software developed to handle the relatively numerous and short genomes of viruses and bacteria.
+Researchers increasingly depend on the outputs of these tools to infer transmission dynamics of human diseases and make actionable recommendations to public health officials [@Black2020; @Gardy2015].
+Under these circumstances, bioinformatics tools must scale rapidly with the number of disease samples to enable real-time analyses of pathogen evolution.
+To meet these needs, we developed Augur, a bioinformatics toolkit designed for phylogenetic analyses of human pathogens.
+
+Augur originally existed as an internal component of the nextflu [@Neher:2015jr] and Nextstrain [@Hadfield2018] applications.
+The original nextflu scripts only supported seasonal influenza viruses.
+When nextflu was replaced with Nextstrain and expanded to support multiple viral and bacterial pathogens, each pathogen received its own copy of the original scripts.
+The resulting redundancy of these large scripts complicated efforts to debug analyses, add new features for all pathogens, and add support for new pathogens.
+In its original form, Augur consisted of two monolithic Python scripts, "prepare" and "process", that performed most operations in memory.
```

Comment:
"In its original form, Augur consisted of a single monolithic Python script, which performed most operations in memory."

Prepare/process came next, as a stepping stone to full modularization :)

Reply:
Good catch! I forgot about this completely.